src/style.css

```diff
@@ -2,0 +1,28 @@
+body {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.stars {
+  display: flex;
+  justify-content: center;
+  flex-direction: row;
+  align-items: center;
+  box-sizing: border-box;
```

Comment:
The `box-sizing: border-box;` declaration is redundant here since it could be set globally for all elements. Consider removing it to avoid repetition.